Shake/Main.lean

```diff
@@ -407,50 +410,60 @@ def main (args : List String) : IO Unit := do
 
   -- Apply the edits to existing files
   if !args.fix then return
-  edits.forM fun mod (remove, add) => do
+  let count ← edits.foldM (fun count mod (remove, add) => do
```

Comment:
```suggestion
  let count ← edits.foldM (init := 0) fun count mod (remove, add) => do
```
I really hate it when the `0` argument is passed 40 lines later

Reply:
That's a good idea! Let me make a follow-up PR.

Reply:
Actually don't worry about it, I've incorporated this into #9751